packages/node-cli/src/commands/AddDependenciesCommandGenerator.js

```diff
@@ -69,18 +69,27 @@ module.exports = class AddDependenciesCommandGenerator extends BaseCommandGenera
 		return answers;
 	}
 
-	_executeAction(answers) {
-		const executionContext = new SDKExecutionContext({
-			command: this._commandMetadata.sdkCommand,
-			params: answers,
-			flags: [COMMAND_OPTIONS.ALL],
-			requiresContextParams: true
-		});
-
-		return executeWithSpinner({
-			action: this._sdkExecutor.execute(executionContext),
-			message: TranslationService.getMessage(MESSAGES.ADDING_DEPENDENCIES),
-		});
+	async _executeAction(answers) {
+		try {
+			const executionContext = new SDKExecutionContext({
+				command: this._commandMetadata.sdkCommand,
+				params: answers,
+				flags: [COMMAND_OPTIONS.ALL],
+				requiresContextParams: true
+			});
+
+			const operationResult = await executeWithSpinner({
+				action: this._sdkExecutor.execute(executionContext),
+				message: TranslationService.getMessage(MESSAGES.ADDING_DEPENDENCIES),
+			});
+
+			return SDKOperationResultUtils.createActionResultFrom(operationResult);
+
+
+			
+		} catch (error) {
+			return new ActionResult.Builder().withError(error).build();
```

Comment:
you can replace `ActionResult.Builder().with...` for `ActionResult.Builder.with...`

Reply:
(In all the occurrences in this PR)